values.yaml

```diff
@@ -3,7 +3,6 @@ images:
   bash: docker.io/library/bash:latest
   skopeo: quay.io/skopeo/stable:latest
   buildah: registry.access.redhat.com/ubi8/buildah:latest
-
```

Comment:
nit: please leave the empty line, they should serve as a logical block separation.